schema.graphql

```diff
@@ -22,9 +22,9 @@ type Protocol @entity {
     "total repaid"
     totalRepaid: BigDecimal
     "threshold amount for Insurance Fund to distribute surplus"
-    insuranceFundFeeDistributionThreshold: BigDecimal!
+    insuranceFundFeeDistributionThreshold: BigDecimal
     "total amount of surplus Insurance Fund has distributed"
-    insuranceFundFeeDistributionTotalAmount: BigDecimal!
+    insuranceFundFeeDistributionTotalAmount: BigDecimal
```

Comment:
This is to merge a hotfix back form `main` to deal with forward-compatibility issue